src/pkg/artifactrash/dao/dao.go

```diff
@@ -71,9 +72,9 @@ func (d *dao) Filter(ctx context.Context, cutOff time.Time) (arts []model.Artifa
 		return deletedAfs, err
 	}
 
-	sql := `SELECT aft.* FROM artifact_trash AS aft LEFT JOIN artifact af ON (aft.repository_name=af.repository_name AND aft.digest=af.digest) WHERE (af.digest IS NULL AND af.repository_name IS NULL) AND aft.creation_time <= ?`
+	sql := fmt.Sprintf(`SELECT aft.* FROM artifact_trash AS aft LEFT JOIN artifact af ON (aft.repository_name=af.repository_name AND aft.digest=af.digest) WHERE (af.digest IS NULL AND af.repository_name IS NULL) AND aft.creation_time <= TO_TIMESTAMP('%f')`, float64(cutOff.UnixNano())/float64((time.Second)))
 
-	_, err = ormer.Raw(sql, cutOff).QueryRows(&deletedAfs)
+	_, err = ormer.Raw(sql).QueryRows(&deletedAfs)
```

Comment:
Is there a better way other than `Sprintf` the SQL?

Doesn't `orm` support comparing time?

Reply:
orm cannot handle comparing time in milliseconds level.

Reply:
use prepared statements?